src/praisonai-code/praisonai_code/cli/commands/run.py

```
@@ -1482,6 +1484,46 @@ class Args:
+    """
+    allow_list = None
+    if subagents:
+        allow_list = [name.strip() for name in subagents.split(",") if name.strip()]
+
+    try:
+        from praisonai_code.cli.features.custom_definitions import (
+            build_subagent_resolver,
+        )
+
+        resolver, descriptions = build_subagent_resolver(allow_list)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)